### PR TITLE
Send "vote-safe" type when using Gnosis safe and show some other noti…

### PR DESCRIPTION
**Pull request by samuveth**
@bonustrack any idea what notification we should show when voting with safe?

### CODE REVIEW

File: src/components/Modal/Confirm.vue [hunk 1]
```diff
@@ -17,17 +18,20 @@ const props = defineProps({
...
 );
-
-const format = getChoiceString;
+const walletConnectType = computed(() => web3.value.walletConnectType);
 
 async function handleSubmit() {
   loading.value = true;
-  await send(props.space.id, 'vote', {
+  const isSafe = walletConnectType.value === 'Gnosis Safe Multisig';
+  await send(props.space.id, isSafe ? 'vote-safe' : 'vote', {
```
Comment (bonustrack) [inline src/components/Modal/Confirm.vue:34]: I've added a function "client.sign" in the client that does exactly like "client.broadcast" but without sending the tx. Can you try to use this function instead?

Comment (samuveth) [reply src/components/Modal/Confirm.vue:34]: Ok, @ChaituVR can you release a new version for this so I can use it?

**Comment by bonustrack** [comment]
@samuveth Yes that's the idea, is there a way you can reduce duplicate code? You could just use something like
```const fn = isSafe? sign : broadcast;
fn('vote', ...
```

**Comment by samuveth** [comment]
> @samuveth Yes that's the idea, is there a way you can reduce duplicate code? You could just use something like
> 
> ```
> const fn = isSafe? sign : broadcast;
> fn('vote', ...
> ```

Do you also see a way to reduce code in useClient?

File: src/components/Modal/Confirm.vue [hunk 2]
```diff
@@ -16,18 +17,22 @@ const props = defineProps({
...
   props.strategies.map(strategy => strategy.params.symbol)
 );
-
-const format = getChoiceString;
+const walletConnectType = computed(() => web3.value.walletConnectType);
 
 async function handleSubmit() {
   loading.value = true;
-  await send(props.space.id, 'vote', {
+  const isSafe = walletConnectType.value === 'Gnosis Safe Multisig';
```
Comment (bonustrack) [inline src/components/Modal/Confirm.vue:33]: This should be done in useClient, we want the same logic for create a proposal and update space settings

Comment (samuveth) [reply src/components/Modal/Confirm.vue]: Ok makes sense

**Comment by samuveth** [comment]
@bonustrack this still requires the latest version of `snapshot.js` that doesn't have a version yet.

**Review by bonustrack — APPROVED** [review]

**Comment by samuveth** [comment]
@bonustrack making votes show in results after signed by safe owners and removing the "Can't sign with Gnosis safe" modal we do in another PR?

**Comment by bonustrack** [comment]
@samuveth What you mean with "making votes show in results after fully signed by safe owners"?

>  removing the "Can't sign with Gnosis safe" modal we do in another PR?

I created an issue for this #830

> "Your vote is in" also doesn't seem to be the right text here.

I created an issue for this #829

**Comment by samuveth** [comment]
> @samuveth What you mean with "making votes show in results after fully signed by safe owners"?

I mean because votes from a Safe aren't stored, how do they ever appear in the results?

**Pull request merged**